### ISSUE TITLE
dagsync incorrectly identifies Panorama URL as backup HA device ...

### ISSUE THREAD

**Issue by agwilkinson** [brian1917/workloader#78]
H all,

Currently using workloader(1) version 10.0.1 to test dag-sync feature.

Running dag-sync against Panorama (latest) _10.2.4-h4 (27 Jul 2023)_ I always get the below error regardless whether I point my URL to the active or passive Panorama node i.e.

2023-08-08 14:00:57 -----------------------------------------------------------------------------
2023-08-08 14:00:57 [INFO] - workloader version 10.0.1 - started PanOS DAG Sync - change=true, insecure=true, ipv6=false, flush=false, rmeoveOld=true
2023-08-08 14:00:57 [INFO] - using default pce - default-pce - 23.3.0-1483
2023-08-08 14:00:57 [ERROR] - URL entered is trying to use backup HA device. URL - https://foo.bar.com

The command I run is:

$ ./workloader --debug --verbose dag-sync --update-panos -i -u https://foo.bar.com -k my-key -f dag-sync.csv

**Comment by agwilkinson** [comment]
Any movement on this one ?

**Comment by jdschmitz15** [comment]
high.   Can you provide the output to the following show command via the API.  It should ask you to login.   There should be a submit button.  It should provide something like the following:   

<response status="success">
<result>
<enabled>no</enabled>
<local-info>
<encrypt-imported>no</encrypt-imported>
</local-info>
</result>
</response>

What matters is what is showing up in between <enabled>field</enabled>.  I expect "active" or "active-primary".  From the docs it might be "active-controller".  The output should verify what Panorama returns for show high-availability state.

https://52.14.221.219/php/rest/browse.php/op::show::high-availability::state

**Comment by agwilkinson** [comment]
<response status="success">
<result>
<enabled>yes</enabled>
<local-info>
<version>1</version>
<state>secondary-passive</state>
<state-duration>827397</state-duration>
<last-error-reason>User requested</last-error-reason>
<last-error-state>secondary-suspended</last-error-state>
<mgmt-ip>10.1.1.1/24</mgmt-ip>
<mgmt-ipv6/>
<preemptive>no</preemptive>
<promotion-hold>2000</promotion-hold>
<hello-interval>8000</hello-interval>
<heartbeat-interval>2000</heartbeat-interval>
<preempt-hold>1</preempt-hold>
<monitor-fail-holdup>0</monitor-fail-holdup>
<addon-master-holdup>7000</addon-master-holdup>
<encrypt-imported>yes</encrypt-imported>
<mgmt-macaddr>00:50:56:90:7e:92</mgmt-macaddr>
<encrypt-enable>no</encrypt-enable>
<link-mon-intv>3000</link-mon-intv>
<priority>secondary</priority>
<build-rel>10.2.4-h3</build-rel>
<url-version>Not Installed</url-version>
<app-version>8744-8227</app-version>
<iot-version>89-421</iot-version>
<av-version>4545-5062</av-version>
<CTS>Match</CTS>
<VMS>Match</VMS>
<build-compat>Match</build-compat>
<url-compat>Match</url-compat>
<app-compat>Match</app-compat>
<iot-compat>Match</iot-compat>
<av-compat>Match</av-compat>
</local-info>
<peer-info>
<conn-ha1>
<conn-status>up</conn-status>
<conn-primary>yes</conn-primary>
<conn-desc>heartbeat status</conn-desc>
</conn-ha1>
<conn-status>up</conn-status>
<version>1</version>
<state>primary-active</state>
<state-duration>827416</state-duration>
<last-error-reason>User requested</last-error-reason>
<last-error-state>primary-suspended</last-error-state>
<preemptive>no</preemptive>
<mgmt-ip>10.1.1.2</mgmt-ip>
<mgmt-macaddr>00:50:56:90:ef:35</mgmt-macaddr>
<priority>primary</priority>
<build-rel>10.2.4-h3</build-rel>
<url-version>Not Installed</url-version>
<app-version>8744-8227</app-version>
<iot-version>89-421</iot-version>
<av-version>4545-5062</av-version>
<CTS>2.0.0</CTS>
<VMS>3.0.4</VMS>
</peer-info>
<path-monitoring>
<enabled>yes</enabled>
<failure-condition>any</failure-condition>
<groups/>
</path-monitoring>
<running-sync>synchronized</running-sync>
<running-sync-enabled>yes</running-sync-enabled>
</result>
</response>

**Comment by agwilkinson** [comment]
```<response status="success">
<result>
<enabled>yes</enabled>
<local-info>
<version>1</version>
<state>secondary-passive</state>
<state-duration>827397</state-duration>
<last-error-reason>User requested</last-error-reason>
<last-error-state>secondary-suspended</last-error-state>
<mgmt-ip>10.230.91.98/24</mgmt-ip>
<mgmt-ipv6/>
<preemptive>no</preemptive>
<promotion-hold>2000</promotion-hold>
<hello-interval>8000</hello-interval>
<heartbeat-interval>2000</heartbeat-interval>
<preempt-hold>1</preempt-hold>
<monitor-fail-holdup>0</monitor-fail-holdup>
<addon-master-holdup>7000</addon-master-holdup>
<encrypt-imported>yes</encrypt-imported>
<mgmt-macaddr>00:50:56:90:7e:92</mgmt-macaddr>
<encrypt-enable>no</encrypt-enable>
<link-mon-intv>3000</link-mon-intv>
<priority>secondary</priority>
<build-rel>10.2.4-h3</build-rel>
<url-version>Not Installed</url-version>
<app-version>8744-8227</app-version>
<iot-version>89-421</iot-version>
<av-version>4545-5062</av-version>
<CTS>Match</CTS>
<VMS>Match</VMS>
<build-compat>Match</build-compat>
<url-compat>Match</url-compat>
<app-compat>Match</app-compat>
<iot-compat>Match</iot-compat>
<av-compat>Match</av-compat>
</local-info>
<peer-info>
<conn-ha1>
<conn-status>up</conn-status>
<conn-primary>yes</conn-primary>
<conn-desc>heartbeat status</conn-desc>
</conn-ha1>
<conn-status>up</conn-status>
<version>1</version>
<state>primary-active</state>
<state-duration>827416</state-duration>
<last-error-reason>User requested</last-error-reason>
<last-error-state>primary-suspended</last-error-state>
<preemptive>no</preemptive>
<mgmt-ip>10.230.91.99</mgmt-ip>
<mgmt-macaddr>00:50:56:90:ef:35</mgmt-macaddr>
<priority>primary</priority>
<build-rel>10.2.4-h3</build-rel>
<url-version>Not Installed</url-version>
<app-version>8744-8227</app-version>
<iot-version>89-421</iot-version>
<av-version>4545-5062</av-version>
<CTS>2.0.0</CTS>
<VMS>3.0.4</VMS>
</peer-info>
<path-monitoring>
<enabled>yes</enabled>
<failure-condition>any</failure-condition>
<groups/>
</path-monitoring>
<running-sync>synchronized</running-sync>
<running-sync-enabled>yes</running-sync-enabled>
</result>
</response>```

**Comment by agwilkinson** [comment]
bugfix PR submitted.

**Comment by brian1917** [comment]
resolved completely in 10.0.4
